packages/compass-explain-plan/src/components/explain-tree/explain-tree-stage.tsx

```diff
@@ -71,27 +86,85 @@ const separatorStyles = css({
 const contentStyles = css({ position: 'relative' });
 const clockStyles = css({
   position: 'absolute',
-  top: -(12 + spacing[2]),
-  right: -(30 + spacing[2]),
+  top: -(8 + spacing[5]),
+  right: -(30 + spacing[3]),
 });
 
 const codeContainerStyles = css({
-  marginTop: spacing[2],
-});
-
-const codeStyles = css({
-  maxHeight: spacing[5] * 10,
-  overflow: 'scroll',
+  marginTop: spacing[3],
+  height: '100%',
+  overflow: 'hidden',
 });
 
 const executionStatsStyle = css({
   position: 'relative',
   display: 'grid',
   gridTemplateColumns: '1fr 1fr',
+  marginTop: spacing[3],
+  alignItems: 'center',
+});
+
+const statsBadgeCircle = css({
+  display: 'inline-block',
+  width: spacing[4] - 4,
+  height: spacing[4] - 4,
+  lineHeight: `${spacing[4] - 4}px`,
+  borderRadius: '100px',
+  textAlign: 'center',
+  fontWeight: 700,
+
+  backgroundColor: palette.blue.base,
+  color: palette.white,
+});
+
+const statsBadgeOval = css({
+  width: 'auto',
+  padding: `0 ${spacing[2]}px`,
+});
+
+const statsBadgeColorDark = css({
+  backgroundColor: palette.blue.light2,
+  color: palette.black,
+});
+
+const shardViewContainerStyles = css({
+  borderRadius: 0,
+  borderColor: palette.gray.base,
+  width: defaultCardWidth,
+  padding: `${spacing[2]}px`,
```

Comment:
This doesn't seem right, in the designs the total height is 32, here it's 46 due to line height + padding

![image](https://github.com/mongodb-js/compass/assets/5036933/1e1b631e-2f24-458a-b55d-e0808eff5477)